designer/submodules/packages/components/src/components/usages/usagesTable.tsx

```diff
@@ -67,8 +67,8 @@ export function UsagesTable(props: TableViewData<ComponentUsageType>): JSX.Eleme
                 sortingOrder: ["desc", "asc", null],
             },
             {
-                field: "nodesId",
-                headerName: t("table.usages.title.NODES_ID", "Nodes"),
+                field: "nodesMetadata",
+                headerName: t("table.usages.title.NODES_METADATA", "Nodes"),
```

Comment:
Why did you change `table.usages.title.NODES_ID`? This placeholder hasn't been changed.